pkgs/applications/networking/cluster/kubectl-ktop/default.nix

```
@@ -0,0 +1,40 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "ktop";
+  version = "0.3.0";
```

```suggestion
  version = "0.3.5";
```

https://github.com/vladimirvivien/ktop/releases/tag/v0.3.5 is out now (don't forget to update both hashes)